transformation/Makefile

```diff
@@ -0,0 +1,30 @@
+DBS = "bq/"
+LIBRARY = "library/"
+ALL_SUBFOLDERS = $(DBS) $(LIBRARY)
+
+ENABLED_BQ ?= 1
+ENABLED_SF ?= 0
+
+TARGET = quadkey_library.js
```

Comment:
No quadkey. As I commented if we want to have different modules not just turf we can maybe call it transformation_library.js